string question
string[] possible_answers
---
bool question_asked
---
string answer